addon/mixins/adapter-fetch.js

```diff
@@ -0,0 +1,189 @@
+import Ember from 'ember';
+import fetch from 'fetch';
+
+const {
+  assign,
+  merge,
+  RSVP
+} = Ember;
+
+/**
+ * Helper function that turns the data/body of a request into a query param string.
+ * This is directly copied from jQuery.param.
+ * @param {Object} queryParamsObject
+ * @returns {String}
+ */
+export function serialiazeQueryParams(queryParamsObject) {
+  var s = [], rbracket = /\[\]$/;
```

Comment:
rbracket should likely be `const RRACKET = /...` at the top of the module (no need to redefine it.